cmd/boostd/backup.go

```diff
@@ -363,26 +306,47 @@ var restoreCmd = &cli.Command{
 	},
 }
 
-func copyFiles(src, dest string) error {
-	f, err := os.Stat(src)
-
-	if os.IsNotExist(err) {
-		fmt.Printf("Not copying %s as file does not exists\n", src)
-		return nil
-	}
+// This function uses chdir() to deal with relative symlinks be careful when reusing
+func copyFiles(srcDir, destDir string, flist []string) error {
 
-	if err != nil && !os.IsNotExist(err) {
+	if err := os.Chdir(srcDir); err != nil {
 		return err
 	}
```

Comment:
I believe it should be possible to write a relative symlink without changing the current working directory.
See https://stackoverflow.com/questions/35541589/creating-a-relative-symbolic-link-through-the-os-package

Reply:
It wasn't working for me for some reason. Probably due to some mistake in the code. I have removed the chdir() and tested the binary. It seems to be working properly. I am keeping the new copyFiles() structure as this is more portable for reuse.